docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,95 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
+# Running Calico Node Containers as a Service
+
+This guide explains how to run Calico as a system process or service. We
+include examples for Systemd, but the commands can be applied to other init
+daemons such as upstart.
+
+## Running the Calico Node Containers as Systemd Services
+Included here are two sample systemd service files to start each Calico node image as a service.
+
+`calico-node.service`
+
+```
+[Unit]
+Description=calico-node
+After=docker.service
+Requires=docker.service
+
+[Service]
+ExecStartPre=@-/usr/bin/docker rm -f calico-node
+ExecStart=/usr/bin/calicoctl node
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Be sure to update the ETCD_AUTHORITY environment variable to point at the
+correct location where your Etcd datastore is running.
+
+Also, depending on how you've installed Docker, the name of the Docker service
+under the `[Unit]` section may be different (such as being docker-engine.service).
+Be sure to check this before starting the service.
+
+## Starting `calico/node-libnetwork` on its own
+If you are running Calico as a Docker network plugin, it is also necessary to
+have a separate job for handling the Calico libnetwork driver.
+
+You can start the `calico/node-libnetwork` image directly with the `docker run`
+command:
+
+```
+docker run -d --privileged --net=host \
+    -v /run/docker/plugins:/run/docker/plugins \
+    --name=calico-libnetwork -e ETCD_AUTHORITY=localhost:2379
+    calico/node-libnetwork
+```
+
+In this example,
+
+ - Privileged is required since the container creates network devices.
+ - Host network is used since the network changes need to occur in the host namespace
+ - The /run/docker/plugins volume is used to allow the plugin to communicate with Docker.
+
+#### Systemd service example
+
+`calico-libnetwork.service`
+
+```
+[Unit]
+Description=calico-libnetwork
+After=docker.service
+Requires=docker.service
+
+[Service]
+ExecStartPre=@-/usr/bin/docker rm -f calico-libnetwork
+ExecStart=/usr/bin/docker run -d --privileged --net=host -v /run/docker/plugins:/run/docker/plugins --name=calico-libnetwork -e ETCD_AUTHORITY=localhost:2379 calico/node-libnetwork:latest
```

Comment:
Need to mentioned to adjust ETCD_AUTHORITY, or perhaps have this reference an environment variable and use same technique as calico node job.